stock_inventory/tests/test_stock_inventory.py

```diff
@@ -292,6 +292,8 @@ def test_05_category_selection(self):
             }
         )
         inventory1.action_state_to_in_progress()
+        # Remove company_id from stock-quant to test it works as expected
+        inventory1.stock_quant_ids.write({"company_id": False})
```

Comment:
@AlexPForgeFlow In fact, functionally, this case correspond to a shared location between several companies (as company_id is a related on quant location).

I would have added this explanation here and in the commit message (that is too poor)